.travis.yml

```diff
@@ -9,3 +9,7 @@ smalltalk:
   - Squeak-5.2
   - Squeak-5.1
   - Squeak-5.0
+
+matrix:
+  allow_failures:
+    - smalltalk: Squeak-5.0
```

Comment:
Mir ist nicht ganz ersichtlich was genau ihr hier verändert habt. So wie ich das interpretiere erlaubt ihr, dass die Test für 5.0 fehlschlagen können. Liege ich da richtig ?

Reply:
Die »Build-Matrix« ist bei Travis die Kombination von Umgebungen/Betriebssystemen, unter denen das Projekt getestet wird – in unserem Fall die sechs Kombinationen aus Betriebssystemen und Smalltalk-Versionen, die implizit ein paar Zeilen vorher definiert sind. Die Matrix kann theoretisch aber noch mehr Dimensionen haben. 

`allow_failures` tut genau, was du vermutet hast. Es erlaubt fehlschlagende Tests für bestimmte Kombinationen, in diesem Fall alle, bei denen die Smalltalk-Version eben Squeak 5.0 ist.

Doku: https://docs.travis-ci.com/user/build-matrix/#rows-that-are-allowed-to-fail

Reply:
@tillprochaska danke für die super Erklärung :) . @linusha hat Eva denn erlaubt, dass die Tests auf Squeak 5.0 fehlschlagen dürfen ? Die Methode `IPTReportTestCase >> testChangeSourceCode` funktioniert ja ebenfalls nicht auf Squeak 5.0 und ich würde den Test gerne nicht mehr auskommentieren, da dieser relativ wichtig ist.

Reply:
Ja, Eva hatte das grundsätzlich erlaubt. Mit der oben beschriebenen Einstellung ist ja trotzdem auch ersichtlich, ob eine Version auf 5.0 funktioniert.
Wird aber bitte einmal einen Blick in die Commit-History, mit dem umgeschriebenen Test ist 5.0 wieder voll funktionsfähig, daher habe ich Travis wieder so eingestellt wie es vorher war.